01_homework/hw2_task5.cpp

```diff
@@ -1,60 +1,74 @@
 #include <iostream>
 #include <chrono>
 #include <array>
+#include <vector>
 #include <thread>
 #include <numeric>
 
-constexpr int reps = 1000000;
+constexpr int reps = 2000000;
 
 void stress_function(int a, int &b) {
     for(int i = 0; i < reps; i++) {
         b = a^b;
     }
 }
 
+struct AlignedInt {
+    alignas(64) int value[6];
```

Comment:
problem nie znikł, bo całą tablicę wyrównujesz do 64. Musiałbyć po prostu inta wyrównać.
Możesz zrobić tak, że tutaj po prostu jest 6 luźnych intów albo utworzyć sobie strukturę typu Int64, która ma 1 inta wyrównanego i taką tablicę Int64[] potestować.

Reply:
Dodałem klase Int64 i przeciazylem odpowiedni operator ale dziala to jeszcze wolniej :/